megamek/src/megamek/client/ratgenerator/ChassisRecord.java

```diff
@@ -45,6 +48,13 @@ public void addModel(ModelRecord model) {
         return models;
     }
     
+    public List<ModelRecord> getSortedModels() { 
+        List<ModelRecord> sortedModels = new ArrayList<>(models);
+        Collections.sort(sortedModels);
```

Comment:
This sort does nothing. ModelRecord::compareTo sorts by the chassis name. All the models handled here have the same chassis and considered equal.

```suggestion
        sortedModels.sort(Comparator.comparing(ModelRecord::getModel));
```